Clients/GUI/App.razor

```diff
@@ -1,4 +1,4 @@
-﻿<Router AppAssembly="@typeof(Program).Assembly">
+<Router AppAssembly="@typeof(Program).Assembly">
     <Found Context="routeData">
         <AuthorizeRouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)">
             <Authorizing> Signing in... </Authorizing>
```

Comment:
is this what will be seen by users? Why "..."?

Reply:
> is this what will be seen by users? Why "..."?

This will be shown to user if the authorization is in process when user wants to access a page, which depends on current state of authorization

Reply:
Ok